angular/projects/spark-angular/src/lib/components/inputs/sprk-checkbox.stories.ts

```diff
@@ -65,7 +65,7 @@ export default {
       ],
     }),
     componentWrapperDecorator(
-      (story) => `<div class="sprk-o-Box">
+      (story) => `<div>
```

Comment:
Do we need these empty `<div>` elements anymore if we aren't putting a class on them?